sdk/servicebus/service-bus/test/README.md

```diff
@@ -1,21 +1,18 @@
-## Pre-requisites
+# Testing
 
-- Run `npm i` to install all the dependencies of this project. This is a one time task.
-- The tests expect a Service Bus namespace to exist.
-  The connection string for the service bus should be in the environment variable `SERVICEBUS_CONNECTION_STRING`. 
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
 
-  Note that the tests will recreate entities to get a clean start before running each test.
+The environment variables you will need to properly assign to run the live test of this project are the following:
 
-  See `testUtils.ts` file for information on names used for the entities against which the tests are run.
+- `AZURE_CLIENT_ID`: The Client ID of your Azure account.
+- `AZURE_CLIENT_SECRET`: The secret of your Azure account.
+- `AZURE_TENANT_ID`: The Tenant ID of your Azure account.
```

Comment:
I dont believe that the descriptions above for client id/secret and tenant id are accurate. These are not properties of an "Azure account"

Reply:
I'll change it to this:

```
- `AZURE_CLIENT_ID`: The client ID of an Azure Active Directory application.
- `AZURE_CLIENT_SECRET`: The client secret of an Azure Active Directory application.
- `AZURE_TENANT_ID`: The Tenant ID of your organization in Azure Active Directory.
```